components/layout/stylesheets/details.css

```diff
@@ -0,0 +1,18 @@
+/* We should use `content-visibility` but it is yet to be implemented.
+ * These rule would not comply with ::details-content and should be removed
+ * with it's implementation */
+slot:not([name]) {
+  display: none;
+}
+:host([open]) slot:not([name]) {
+  display: block;
+}
+
+summary[name=internal-fallback-summary] {
```

Comment:
I think this should be applied to anything slotted into the summary slot, not just the fallback summary.

So the selector should be `#internal-summary::slotted(summary)`, after changing to use `id` to mark a specific slot.

Reply:
I have modified the corresponding stylesheet, please kindly check!